point-of-care-poc/work-flows/hiv-clinic.md

```diff
@@ -0,0 +1,64 @@
+### **HIV Clinic**
```

Comment:
@slubwama Remove the bold from each of the titles to be consistent with the rest of the documentation